integration-tests/tests/package.json

```diff
@@ -39,7 +39,8 @@
   "dependencies": {
```

Comment:
the package-lock.json diff above seems quite big, could it be the case that I've done something weird here?